src/docbuilder/metadata/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "metadata"
+version = "0.1.0"
+authors = ["Joshua Nelson <jyn514@gmail.com>"]
+edition = "2018"
+license = "MIT"
+repository = "https://github.com/rust-lang/docs.rs"
+description = "Document crates the same way docs.rs would"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+path = "lib.rs"
+
+[dependencies]
+toml = { version = "0.5", default-features = false }
+thiserror = "1"
+
+[package.metadata.docs.rs]
+rustdoc-args = ["--document-private-items"]
```

Comment:
Is there a reason why? People would mistake that for the crate's public API.

Reply:
It's left over from when the docs on `BuildTargets` were still on the private fields. I'll remove it.